### PR TITLE
Displaying which players have finished their turn

### PR DESCRIPTION
**Pull request by sibela46**

### CODE REVIEW

File: server/services/GameLogic.js [hunk 1]
```diff
@@ -239,6 +247,12 @@ module.exports = class GameLogic {
     /* Returns true if all players (apart from the current player) have voted this round */
     allPlayersVoted() { return this.players.every(player => player.username === this.currentPlayer.username || this.playerHasVoted(player.username)) };
 
+    /* Mark the player's turn as finished and notify the other players */
+    markTurnAsFinished() {
+        this.players.find(player => player.username === username).finishedTurn = true;
```
Comment (Ro4052) [inline server/services/GameLogic.js:252]: Where is `username` defined? Is it a global variable?

File: src/components/Players/Players.jsx [hunk 2]
```diff
@@ -7,13 +7,16 @@ const cx = classNames.bind(styles);
 
 export class Players extends React.PureComponent {
     render() {
+        console.log(this.props.players);
```
Comment (Ro4052) [inline src/components/Players/Players.jsx]: Should probably remove this log.

File: server/services/GameLogic.js (hunk 1 above)
Comment (sibela46) [reply server/services/GameLogic.js:252]: oops, that's when I moved it to a function, that's why the tests aren't passing, my bad

File: server/services/socket.js [hunk 3]
```diff
@@ -9,6 +9,8 @@ exports.setupSocket = (server, session) => {
     io = socketio(server);
     io.use(sharedsession(session));
     io.on('connection', function (socket) {
+        console.log("hi");
```
Comment (Ro4052) [inline server/services/socket.js]: Now remove this one?

Comment (sibela46) [reply server/services/socket.js]: oops

**Review by Ro4052 — APPROVED** [review]

**Pull request merged**